runtime/plugin/matchit.vim

```diff
@@ -1,8 +1,10 @@
 "  matchit.vim: (global plugin) Extended "%" matching
-"  Last Change: Fri Jul 29 01:20 AM 2016 EST
+"  Last Change: 2016 Aug 21
 "  Maintainer:  Benji Fisher PhD   <benji@member.AMS.org>
 "  Version:     1.13.2, for Vim 6.3+
 "		Fix from Tommy Allen included.
```

Comment:
I have no idea what's going on with that.  He's been silent for a year then [merges some old PRs](https://github.com/benjifisher/matchit.zip/commits/master) _except_ the one dealing with the [broken jumps](https://github.com/benjifisher/matchit.zip/pull/2).  The PR I've submitted to vim is also silent: https://github.com/vim/vim/pull/955.  I didn't even get the expected reply telling me to contact the maintainer, so I kinda just stopped caring.